keps/sig-storage/5040-remove-gitrepo-driver/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: Disable hooks for gitRepo volumes by default.
+kep-number: 5040
+authors:
+  - "@vinayakankugoyal"
+owning-sig: sig-storage
+participating-sigs:
+status: provisional
+creation-date: 2024-12-14
+reviewers:
+  - @thokin
```

Comment:
"ck"

Reply:
I think you can make me the approver along with someone from sig-node -- @SergeyKanzhelev can you nominate someone?  Maybe make @saad-ali a reviewer too?

Reply:
Done. Thanks!